libs/membership/website/src/lib/subscribe/subscribe-container.tsx

```diff
@@ -37,6 +39,7 @@ export const SubscribeContainer = <T extends OptionalKeysOf<RegisterMutationVari
   schema,
   filter
 }: SubscribeContainerProps<T>) => {
+  const router = useRouter()
```

Comment:
This has to be outside the website builder.

Reply:
That means, I have to pass it through the properties? Any other idea?

Reply:
Apollo Links => Error Links